### PR TITLE
[ADD] DCOS-367 As a user, I want to scale the application

### PR DESCRIPTION
**Pull request by opolishchuck**

### CODE REVIEW

**Comment by ssk2** [comment]
@opolishchuck - please rebase this feature on master!

**Comment by opolishchuck** [comment]
@ssk2 done.

**Comment by ssk2** [comment]
@opolishchuck  - apologies. That was not an approval to merge. I have a few comments to make. Please address these (you may need to submit a new PR).

File: cli/dcoscli/marathon/main.py [hunk 1]
```diff
@@ -94,6 +96,8 @@
                                 stdin.
 
     <task-id>                   The task id
+
+    <scale-factor>              The value of scale-factor for a group
```
Comment (ssk2) [inline cli/dcoscli/marathon/main.py]: `The value of scale-factor for a group` -> `The factor to scale an application group by.`

Comment (ssk2) [inline cli/dcoscli/marathon/main.py]: We should also change line 92 above from `The number of instances to start` to `The number of instances to run`

File: cli/dcoscli/marathon/main.py [hunk 2]
```diff
@@ -622,6 +636,44 @@ def _update(app_id, properties, force):
     return 0
 
 
+def _scale(app_id, instances, force):
+    """
+    :param app_id: the id of the application
+    :type app_id: str
+    :param instances: amount of new instances
```
Comment (ssk2) [inline cli/dcoscli/marathon/main.py]: `amount` -> `number`

File: cli/dcoscli/marathon/main.py [hunk 3]
```diff
@@ -622,6 +636,44 @@ def _update(app_id, properties, force):
...
+    deployment = client.scale_app(app_id, instances, force)
+    emitter.publish('Created deployment {}'.format(deployment))
+    return 0
+
+
+def _group_scale(group_id, scale_factor, force):
+    """
+    :param group_id: the id of the group
+    :type group_id: str
+    :param scale_factor: scale factor for group instances
```
Comment (ssk2) [inline cli/dcoscli/marathon/main.py]: `group instances` -> `application group`

File: cli/dcoscli/marathon/main.py [hunk 4]
```diff
@@ -12,6 +12,7 @@
     dcos marathon app start [--force] <app-id> [<instances>]
     dcos marathon app stop [--force] <app-id>
     dcos marathon app update [--force] <app-id> [<properties>...]
+    dcos marathon app scale [--force] <app-id> <instances>
```
Comment (jsancio) [inline cli/dcoscli/marathon/main.py]: Please remove this command. We are using the save verb (`scale`) to mean two different things. The user is already able to to `dcos marathon app update <app-id> instances=<instances>`.

**Pull request merged**